go/cmd/vttablet/vttablet.go

```diff
@@ -87,6 +88,8 @@ func main() {
 	if *tableAclConfig != "" {
 		tableacl.Register("simpleacl", &simpleacl.Factory{})
 		tableacl.Init(*tableAclConfig)
+	} else if *enforceTableACLConfig {
+		panic("there is no valid tableacl config file")
```

Comment:
Do be consistent with the rest of the code, you should probably use exit.Return instead? See code above.

I also suggest to make the error clearer to the user, so he knows what he did wrong:

    log.Error("tableacl config has to be specified with -table-acl-config because -enforce-tableacl-config is set.")
    exit.Return(1)